src/actions/tests/test_radio_interface.py

```diff
@@ -0,0 +1,233 @@
+""" Test aspects of RadioInterface """
+
+from os.path import join
+from actions import usrp
+
+
+class TestRadioInterface(object):
```

Comment:
General layout change here... we're using `pytest` as our primary test runner, which is function based. So remove this class, make the couple class variables you have in here file-level "consts", and then just dedent all your methods to top level and remove `self`... lastly, instead of `raise Exception` to indicate a failed test, just use `assert`, and you'll be compatible with pytest.

You can still "return" a custom error message with `assert` using the format `assert condition, msg`

Reply:
Adjusted my tests to better use the built-in functionality of pytest
`with pytest.raises(Exception,...):`
`assert (condition), msg`